net/pfSense-pkg-freeradius3/files/usr/local/pkg/freeradius.inc

```diff
@@ -1601,18 +1601,22 @@ EOD;
 	}
 }
 
+/* This change cannot be pulled back to RELENG_2_4_4 since it would break there.
+ * see https://redmine.pfsense.org/issues/9897
+ */
 // Gets started from freeradiuseapconf.xml
-function freeradius_get_ca_certs() {
-	global $config;
-	$ca_arr = array();
-	$ca_arr[] = array('refid' => 'none', 'descr' => 'none (auto)');
-
-	if (is_array($config['ca'])) {
-		foreach ($config['ca'] as $ca) {
-			$ca_arr[] = array('refid' => $ca['refid'], 'descr' => $ca['descr']);
-		}
+function freeradius_get_ca_or_certs($type) {
+	$c_arr = array();
+	if ($type == 'ca') {
+		$ecdsagood = cert_build_list('ca', 'IPsec');
+	} else {
+		$ecdsagood = cert_build_list('cert', 'IPsec');
+		$c_arr[] = array('refid' => 'none', 'descr' => 'none (auto)');
```

Comment:
Why is this only added to the certificate array? Before it was added to both.

Reply:
yeap, fixed